ConwaysGameOfLife/InputParser.cs

```diff
@@ -19,6 +19,18 @@ public List<Location> ParseListOfLocationsFromUserInput(string userInput)
             return liveCellLocations;
         }
 
+        public Universe ParseUniverse(string userInput)
+        {
+            var universe = SplitInput(userInput, ",");
+            return new Universe(ParseInput(universe[0]), ParseInput(universe[1]));
+        }
+
+        public Location ParseLocation(string input)
+        {
+            var location = SplitInput(input, ",");
+            return new Location(ParseInput(location[0]), ParseInput(location[1]));

```

Comment:
should we be validating before or after using the input? i.e. do we want to attempt to create the universe if the input isn't valid?